assets/sass/v2/_duo.scss

```diff
@@ -0,0 +1,7 @@
+.duo-authenticator-enroll,
+.duo-authenticator-challenge {
+  iframe {
```

Comment:
Looking at the screenshots... this looks just slightly centered to the left. Can we force alignment to the center here?

Reply:
I am not changing anything with the padding, it should be the same in both left and right. It is coming from [_layout.scss](https://github.com/okta/okta-signin-widget/blob/master/assets/sass/_layout.scss#L7)